router/analytics.go

```diff
@@ -56,6 +56,10 @@ func (a *Analytics) run() {
 			continue
 		}
 
-		_, _ = http.Post(a.url, "application/json", bytes.NewReader(msgbytes))
+		resp, err := http.Post(a.url, "application/json", bytes.NewReader(msgbytes))
+		if err == nil && resp != nil && resp.Body != nil {
```

Comment:
Nitpick, from Go source (https://golang.org/src/net/http/client.go) `When err is nil, resp always contains a non-nil resp.Body.` - so checking `resp.Body` is not nil is redundant.